docs/website/blog/2023-02-15-odo-v3.7.0.md

```diff
@@ -28,8 +28,6 @@ Similarly to how `odo dev` requires specifying the target platform (using the `-
 ODO_EXPERIMENTAL_MODE=t odo logs --platform podman
 ```
 
-As a reminder, Podman support in `odo` is still [experimental](../docs/user-guides/advanced/experimental-mode) and requires you to [opt in](../docs/user-guides/advanced/experimental-mode#enabling-the-experimental-mode) if you want to try it out.
```

Comment:
I would keep this as is since at the time of v3.7.0 podman was still in experimental mode.

Reply:
I'm reverting it, and only removing links